src/query/executor.rs

```diff
@@ -1,14 +1,20 @@
-use crate::ff::{Field, FieldType, Fp31};
-use crate::helpers::messaging::Gateway;
-use crate::helpers::query::{IPAQueryConfig, QueryConfig, QueryType};
-use crate::helpers::{negotiate_prss, TransportError};
-use crate::protocol::attribution::AggregateCreditOutputRow;
-use crate::protocol::context::SemiHonestContext;
-use crate::protocol::ipa::{ipa, IPAInputRow};
-use crate::protocol::Step;
-use crate::secret_sharing::Replicated;
-use crate::task::JoinHandle;
-use futures::Stream;
+use crate::{
```

Comment:
you need to teach me how to do it - Clion puts every import on a separate line and `cargo fmt` does not fix it for me

Reply:
sorry to report that I am doing it manually lol

Reply:
omg, don't :) you're gonna lose this battle against autoformatters